qubes/ext/pci.py

```diff
@@ -227,6 +228,9 @@ def on_device_pre_attached_pci(self, vm, event, device):
             return
 
         try:
+            device = next(
+                self.on_device_get_pci(vm, event, device.ident)
+            )
```

Comment:
If so, [dropping `dev.options`](https://github.com/QubesOS/qubes-core-admin/pull/98#discussion_r111396748) is even more important.